src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -278,7 +287,7 @@ void ConnectToStadiaWidget::CheckCredentialsAvailable() {
   }
 
   emit ReadyToDeploy();
-}
+}  // namespace orbit_session_setup
```

Comment:
This is an accident.

Reply:
What exactly do you mean, the comment?

Reply:
Yes. The namespace does not end there. It is an accident, because my best guess on what happened:
While implementing the was accidentally an `}` too much and then clang-format auto inserted this namespace comment there. After the `}` mistake was fixed, clang-format does not remove the comment automatically.

Reply:
Turns out I was looking at the wrong line, fixed, thanks!